src/main/webapp/app/overview/course-statistics/course-statistics.component.html

```diff
@@ -1,5 +1,16 @@
 <div *ngIf="course">
     <div class="row" *ngIf="course.exercises && course.exercises.length > 0 && groupedExercises.length > 0; else noStatistics">
+        <div class="col-8 mt-3 position-relative">
+            <div class="exercise-controls">
+                <label class="d-flex align-items-center mb-0 me-2">
+                    <input type="checkbox" [checked]="currentlyHidingExcludedExercises" (change)="toggleNotScoreRelevantExercises()" />
```

Comment:
Having different names here (hidingExcluded and NotScoreRelevant) might be confusing, can you change both to "notScoreRelevantExercises"?